MIGRATION.md

```diff
@@ -2,6 +2,73 @@
 
 These guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `2.8.0` -> `3.0.0`
+
+### Breaking changes
+
+- Removed support for `buttonId`. From this version you will need to create a function that launches the SDK when a trigger element (ie a button) is clicked.
+
+### Example of old behaviour
+```html
+<script>
+    Onfido.init({
+      useModal: true,
+      buttonId: 'onfido-btn',
+      token: 'YOUR_JWT_TOKEN',
+      onComplete: function(data) {
+        // callback for when everything is complete
+        console.log("everything is complete")
+      },
+      steps: [
```

Comment:
Minor: to keep the migration more focused you can remove the steps and let it default